package.json

```diff
@@ -21,6 +21,9 @@
     "@angular/platform-browser": "^4.0.0",
     "@angular/platform-browser-dynamic": "^4.0.0",
     "@angular/router": "^4.0.0",
+    "@ng-bootstrap/ng-bootstrap": "^1.0.0-beta.1",
+    "bootstrap": "4.0.0-beta",
```

Comment:
Do you need this? Looks like you are pulling it from the CDN. Maybe the `ng-bootstrap` needs it as a peer dependency.